js/routes/MeteorRoute.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express')
+const router = express.Router()
+const meteorService = require('../services/MeteorService.js')
+
+
+router.get('/', async (req, res) => {
+    try {
+        res.json(await meteorService.getMeteorsData())
+    } catch (error) {
+        res.status(500).send(error)
```

Comment:
Would be better to inspect error - Nasa API can potentially return some error, e.g. too many requests, and 500 doesn't fit in this case. Generally it's bad idea to return 500's error to client, if server can process it